protostar/commands/test/testing_live_logger.py

```diff
@@ -71,12 +68,7 @@ def log(
                             else "GREEN"
                         )
 
-                        progress_bar.write(
-                            test_case_result.display(
-                                self.stdout_on_success
-                                or isinstance(test_case_result, FailedTestCase)
-                            )
-                        )
+                        progress_bar.write(test_case_result.display(True))
```

Comment:
```suggestion
                        progress_bar.write(test_case_result.display(include_stdout_section=True))
```
It's easier to read in that way.